/partol_node:
  ros__parameters:
    initial_point: [0.0, 0.0, 0.0 ]
    target_points: [
        0.0, 0.0, 0.0,
        1.0, 2.0, 3.14,
        -1.0, 2.0, 1.57, 
        -2.0, -2.0, 3.14,
    ]
  
    use_sim_time: false